julia/with_rebuilt.py

```diff
@@ -42,8 +49,17 @@ def maybe_rebuild(rebuild, julia):
         yield
 
 
+@contextmanager
+def ignoring(sig):
+    s = signal.signal(sig, signal.SIG_IGN)
+    try:
+        yield
+    finally:
+        signal.signal(sig, s)
```

Comment:
Can we have a comment about what's going on here?

Reply:
I added a docstring.